models/movement.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const mongoose = require("mongoose");
+
+const movementSchema = new mongoose.Schema({
+  name: {
+    type: String.
+    required: true
+  },
+  description: String,
+  start_year: String,
+  end_year: String,
+  ancestor: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'movement',
+  }
```

Comment:
Two small but significant problems.
1. `_ancestor`, since it's a reference
2. `'Movement'`, not `'movement'` - otherwise, Mongo could get confused and try to grab the wrong collection.